jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -226,11 +229,27 @@ class CawsConnectionProvider : GatewayConnectionProvider {
                                         }
 
                                         if (it == attempts - 1) {
+                                            CodecatalystTelemetry.devEnvironmentWorkflowStatistic(
```

Comment:
`measureTimedValue` would be cleaner

Reply:
Sure, I will keep in mind for next time, Since its a temporary metric, so merging ..